tests/karate/src/test/java/fe/utils/web.feature

```diff
@@ -0,0 +1,43 @@
+Feature: web test
+
+  Background: App Preset
+    * configure driver = { type: 'chrome' }
+    * def driverOptions = karate.toAbsolutePath('file:../../fe1-web/web-build/index.html')
+    # Create and import mock backend
+    * call read('classpath:fe/net/mockbackend.feature')
+    * def backendURL = 'ws://localhost:' + backend.getPort()
+    # Import message filters
+    * call read('classpath:common/net/filters.feature')
+
+    # ================= Page Object Start ====================
+
+
+    # Tab buttons
+    * def tab_home_selector = '{}Home'
+    * def tab_connect_selector = '{}Connect'
+    * def tab_launch_selector = '{}Launch'
+    * def tab_wallet_selector = '{}Wallet'
+
+    # Launch tab
+    * def tab_launch_lao_name_selector = "input[placeholder='Organization name']"
+    * def tab_launch_address_selector = "input[placeholder='Address']"
+    * def tab_launch_create_lao_selector = '{}Launch -- Connect, Create LAO & Open UI'
+
+  Scenario: Setup connection to the backend and complete on the home page
+    Given driver driverOptions
+
+    # The default input function is not consistent and does not work every time.
+    # This replaces the input function with one that just tries again until it works.
+    * def input =
```

Comment:
What if it does not work do we get stuck infinitely in the loop ?

Reply:
Hmmm... it goes into an infinit loop...

I'll add a maximum number of tries :D

Reply:
Done